client/src/components/web-extension/ConnectedSites/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { disconnectFromSite, getConnectedSites } from '@cd/components/hooks/useServiceWorker';
+import _isEmpty from 'lodash-es/isEmpty';
+import _get from 'lodash-es/get';
+import { MiddleTruncatedText } from '@cd/components/Common/MiddleTruncatedText/index';
+import UserIcon from '@cd/assets/image/user-icon.svg';
+import DisconnectIcon from '@cd/assets/image/disconnect-icon.svg';
+
+import './index.scss';
+
+const ConnectedSites = () => {
+    const [currentPublicKeys, setPublicKeys] = useState([]);
+    const [currentSite, setCurrentSite] = useState('');
+
+    useEffect(() => {
+        const getPublicKeys = async () => {
+            const sites = await getConnectedSites();
+            if (_isEmpty(sites)) {
+                return;
+            }
+            const tabs = await chrome.tabs.query({
+                active: true,
+                lastFocusedWindow: true
+            });
+            if (_isEmpty(tabs)) {
+                return;
+            }
+            const [currentTab] = tabs;
+            const currentTabUrl = _get(currentTab, 'url', false);
+            if (!_get(currentTab, 'url', false)) {
+                return;
+            }
+            const url = new URL(currentTabUrl);
+
+            const publicKeys = Object.keys(sites).reduce((curry, publicKey) => {
```

Comment:
Better replace `curry` variable name with something like `result`. I don't see any curried function in this reduce.